exercises/change/change_test.go

```diff
@@ -0,0 +1,48 @@
+package change
+
+import (
+	"reflect"
+	"testing"
+)
+
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Fatalf("Found testVersion = %v, want %v", testVersion, targetTestVersion)
+	}
+}
+
+func TestChange(t *testing.T) {
+	for _, tc := range testCases {
+		actual, err := Change(tc.coins, tc.target)
+		if tc.valid {
+			if err != nil {
+				t.Fatalf("%s : Change(%v, %d): expected %v, got error %s",
+					tc.description, tc.coins, tc.target, tc.expectedChange, err)
```

Comment:
I realized I forgot to ask about #275 here (ensuring the second return value is of type `error`). I'm not sure it's necessary, but we could either add a `var _ error = err` or call `err.Error()` explicitly? Or should we not bother applying #275 here?

I should document #275 somewhere since everyone keeps forgetting it, maybe it's not really necessary if people take it for granted.